modules/nbascore/widget.go

```diff
@@ -50,7 +50,6 @@ func NewWidget(app *tview.Application, pages *tview.Pages, settings *Settings) *
 func (widget *Widget) Refresh() {
 	widget.app.QueueUpdateDraw(func() {
 		widget.nbascore()
-		widget.View.SetTitle(widget.ContextualTitle(widget.Name()))
```

Comment:
This change breaks the title if it has been assigned a numeric hot key (all widgets need to render the title through `ContextualTitle` in case they've got a `FocusChar`).

This only applies to widgets that are `Focusable`, though for future-proofing all widget titles should probably just go through `ContextualTitle`.

Reply:
I thought the general case did this anyway, but I will explicitly test to understand the difference. Thanks for pointing this out!

Reply:
Yeah that makes much more sense. Thanks for looking into this.

Reply:
Note that any calls to `SetTitle` must be wrapped in either `QueueUpdate()` or `QueueUpdateDraw` https://github.com/rivo/tview/wiki/Concurrency